src/AVFoundation/AVAudioUnit.cs

```diff
@@ -1,30 +0,0 @@
-// 
-// AVAudioUnit.cs: Support for AVAudioUnit
-//
-// Authors: Miguel de Icaza (marek.safar@gmail.com)
-//     
-// Copyright 2014 Xamarin Inc.
-//
-
-using System;
-
-using Foundation;
-using CoreFoundation;
-using ObjCRuntime;
-using AudioToolbox;
-
-namespace AVFoundation {
-
-	public partial class AVAudioUnit {
-		public AudioComponentDescription AudioComponentDescription {
-			get {
-#if !XAMCORE_2_0
-				return new AudioComponentDescription (_AudioComponentDescription);
-#endif
-			}
-		}
-
-			       
-	}
-}
-
```

Comment:
This seems to remove the API - but it looks like the file is not included in the build.

Reply:
Yes, that's correct, this file wasn't included in the build.